phylogenetic/rules/merge_sequences_usvi.smk

```diff
@@ -0,0 +1,47 @@
+"""
+This part of the workflow appends USVI data to the main dataset.
+
+REQUIRED INPUTS:
+
+    usvi_sequences  = data/sequences_usvi.fasta
```

Comment:
_not blocking_

Is our thinking to use this docstring-like approach to document each rule file? A brief description of intent seems helpful but detailing the inputs & outputs seems prone to falling out of sync

Reply:
Thanks for taking a look and yes, you are correct! 

The rational is to target the docstring for a high-level summary of the rule file's objectives. Users can delve into specific rule docstrings for more in-depth information. We'd aim at keeping the documented inputs and outputs up to date as part of any future modification, a bit similar to maintaining an API.

Reply:
I would advise reconsidering this. The snakemake code documents the inputs/outputs, duplicating these in the ~docstring is going to fall out of sync. Completely up to you & Jover -- just a suggestion!

Reply:
It's a good point! We were also considering just moving the "append_usvi" rule into "prepare_sequences.smk" to bypass having to track anther file. Do you have a preference for keeping it separate or just merging it among the "prepare_sequences" rules?

Reply:
No preference. In general I'd see the naming of the rules as pathogen-dependent; if the bioinformatician feels it's appropriate to carve certain rules out into a new `.smk` file then I think that's just fine.

Reply:
Yeah, I'm also worried about inputs/outputs getting out of sync, but I'm not sure how else to give people a high level view of how the smaller `.smk` files are connected. 

The goal is for these required inputs/outputs to rarely change since they will become the standard connection points.

Reply:
My reading of the comments in the template's (otherwise-empty) snakemake rule files was that they provided a really nice overview of what the ruleset should accomplish, including the expected inputs and outputs¹, but that the docstring paths would be removed by the author when they actually add the rules.

¹ For the simple case -- for builds using wildcards the author will have to understand that these paths need modifying.